.npmrc

```diff
@@ -1 +1 @@
-registry=https://nexusrepo.antwerpen.be/repository/npm-all
+registry=https://nexusrepo.antwerpen.be/repository/npm-all/
```

Comment:
Moet dit afhankelijk zijn van nexus?

Reply:
Binnenkort niet meer, maar voorlopig wel, want daar staat ook nog de core branding, wat een dependency is.